spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageResourceObject.java

```diff
@@ -114,6 +115,11 @@ private Blob resolve() throws IOException {
 	}
 
 	private Blob createBlob() throws IOException {
+		Resource bucket = getBucket();
+		if (!bucket.exists()) {
```

Comment:
Previous comment on logic change here still stands.

Reply:
Yea I just disagreed that comment. It feels pretty natural to me to have the constructor take care of creating the bucket or not based on the auto-create bool setting.

Reply:
If you're doing auto-create, it should be here, not in the constructor.